Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierOperationQuestion.cshtml.cs

```diff
@@ -0,0 +1,63 @@
+using CO.CDP.Mvc.Validation;
+using CO.CDP.Organisation.WebApiClient;
+using CO.CDP.OrganisationApp.Constants;
+using CO.CDP.OrganisationApp.WebApiClients;
+using Microsoft.AspNetCore.Mvc;
+
+namespace CO.CDP.OrganisationApp.Pages.Supplier;
+
+[AuthorisedSession]
```

Comment:
You can just use [Authorize] attribute unless you need anything from logged in user session object.